tests/benchmark-models/test_petab_benchmark.py

```diff
@@ -252,6 +261,123 @@ def benchmark_problem(request):
     return problem_id, petab_problem, amici_model
 
 
+@pytest.mark.filterwarnings(
+    "ignore:The following problem parameters were not used *",
+    "ignore: The environment variable *",
+    "ignore:Adjoint sensitivity analysis for models with discontinuous ",
+)
+def test_jax_llh(benchmark_problem):
```

Comment:
Move jax-related imports in here, so the remaining tests can be run without all the extra deps?